changelog/std_outbuffer_align.dd

```diff
@@ -0,0 +1,27 @@
+Add custom fill value to OutBuffer class
+    
+If you prepare some data, which is intended for flash programming, you
+want to fill 0xff bytes, because this is the default for unprogrammed
+flash device bytes.
+    
+There might also be other usecases, where you want to fill the alignment
+gaps with some custom value, e.g. when debugging and viewing output data
+in a hex editor. It is easier to spot gaps, when you use a custom value
+like 0x55 or 0xaa.
+    
+Also added unittests to test alignments with default and custom fill
+values.
```

Comment:
```suggestion
```
changelog entries are user-facing, unittests are not relevant to them

Reply:
Fixed.